mopidy_musicbox_webclient/static/js/functionsvars.js

```diff
@@ -343,12 +343,9 @@ function resultsToTables(results, target, uri) {
                     }
                     newalbum = [];
                     newtlids = [];
-                    if (results[i].album) {
-                        coversList.push([results[i].album, i]);
-                    }
                 } //newalbum length
                 if (results[i].album) {
-                    coversList.push([results[i].album, i]);
+                    coversList.push([results[i].uri, i]);
```

Comment:
No, it appears it's not used anywhere else. All other places where we display cover art we seem to re-request the image anyway.